eng/stages/publish.yml

```diff
@@ -17,6 +17,20 @@ stages:
     symbolPublishingAdditionalParameters: "'-warnAsError:$false'"
     publishInstallersAndChecksums: true
 
+    SDLValidationParameters:
+        enable: true
+        params: ' -SourceToolsList @("policheck","credscan")
+        -TsaInstanceURL https://devdiv.visualstudio.com/
```

Comment:
IMO `>-` is nicer, lets you format it without awkward indent and the trailing `'`, e.g.:

https://github.com/dotnet/core-setup/blob/d98d0f7070a6985771054b7a4623eb06dceddb47/eng/stages/publish.yml#L25-L35
```yml
      params: >-
        -SourceToolsList @("policheck","credscan")
        -TsaInstanceURL "$(TsaInstanceURL)"
        -TsaProjectName "$(TsaProjectName)"
        -TsaNotificationEmail "$(TsaNotificationEmail)"
        -TsaCodebaseAdmin "$(TsaCodebaseAdmin)"
        -TsaBugAreaPath "$(TsaBugAreaPath)"
        -TsaIterationPath "$(TsaIterationPath)"
        -TsaRepositoryName "$(TsaRepositoryName)"
        -TsaCodebaseName "$(TsaCodebaseName)"
        -TsaPublish $True
```

http://yaml-multiline.info/ is nice to explore.

Reply:
Nice! Will fix :)